debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9ConstantPoolCommand.java

```diff
@@ -0,0 +1,505 @@
+                            int romMethodCount = foundRomClass.romMethodCount().intValue();
+                            while (romMethodCount > 0) {
+                                if (sameNameAndSignature(romMethod.nameAndSignature(), nameAndSignature)) {
+                                    romInfo = "!j9rommethod " + romMethod.getHexAddress();
+                                    break;
+                                }
+                                romMethod = ROMHelp.nextROMMethod(romMethod);
+                                romMethodCount--;
+                            }
+                        }
+                    }
+
+                    value = String.format("%s.%s:%s #%d",
+                                            className,
+                                            J9UTF8Helper.stringValue(nameAndSignature.name()),
+                                            J9UTF8Helper.stringValue(nameAndSignature.signature()),
+                                            romMethodRef.classRefCPIndex().intValue());
+                } else if (cpType == J9ConstantPool.J9CPTYPE_HANDLE_METHOD) {
+                    cpTypeString = "HandleMethodref";
+                    J9ROMMethodRefPointer romMethodRef = J9ROMMethodRefPointer.cast(romCPP);
+                    if (i < ramCPCount) {
+                        J9RAMMethodRefPointer ramMethodRef = J9RAMMethodRefPointer.cast(ramCPP);
+                        long methodIndex = ramMethodRef.methodIndexAndArgCount().longValue() >> 8;
+
+                        Lookup lookup = MethodHandles.lookup();
+                        Class<?> classPointer = Class.forName("com.ibm.j9ddr.vm29.pointer.generated.J9ClassPointer");
+                        MethodHandle mhTable = null;
+                        int javaVersion = vm.j2seVersion().bitAnd(JavaVersionHelper.J2SE_SERVICE_RELEASE_MASK).intValue() >> JavaVersionHelper.J2SE_JAVA_SPEC_VERSION_SHIFT;
+                        if (javaVersion < 17) {
+                            mhTable = lookup.findVirtual(classPointer, "methodTypes", MethodType.methodType(PointerPointer.class));
+                        } else {
+                            mhTable = lookup.findVirtual(classPointer, "invokeCache", MethodType.methodType(PointerPointer.class));
+                        }
+
+                        PointerPointer mhObject = (PointerPointer)mhTable.invoke(ramClass);
+                        mhObject = mhObject.add(methodIndex);
+
+                        if (mhObject.isNull()) {
+                            ramInfo = "NULL";
+                        } else {
+                            ramInfo = "!j9object " + mhObject.getHexValue();
+                        }
+                    }
+                    J9ROMNameAndSignaturePointer nameAndSignature = romMethodRef.nameAndSignature();
+                    romInfo = "!j9romnameandsignature " + nameAndSignature.getHexAddress();
+                    value = "invoke/invokeExact";
```

Comment:
you should be able to get the name/signature from the ROM info, I think that would be better than having a generic `invoke/invokeExact`